lms/views/api/d2l/authorize.py

```diff
@@ -17,19 +17,22 @@
     permission=Permissions.API,
 )
 def authorize(request):
-    application_instance = request.find_service(
-        name="application_instance"
-    ).get_current()
-    state = OAuthCallbackSchema(request).state_param()
-
-    scopes = ["core:*:*"]
+    scopes = []
 
     if request.product.settings.files_enabled:
         scopes += FILES_SCOPES
 
     if request.product.settings.groups_enabled:
         scopes += GROUPS_SCOPES
 
+    if not scopes:
```

Comment:
We don't want to request a token with no scopes, if we are doing that  something went wrong.